src/fabric_view_changes.erl

```diff
@@ -461,6 +464,13 @@ validate_start_seq(DbName, Seq) ->
             {error, {bad_request, Reason}}
     end.
 
+get_changes_epoch() ->
+    {ok, Epoch} = application:get_env(fabric, changes_epoch),
+    Epoch.
+
+increment_changes_epoch() ->
+    application:set_env(fabric, changes_epoch, get_changes_epoch() + 1).
```

Comment:
That doesn't sound terrible. You could also use the "don't persist" version of `config:set/4` IIRC.